django_webapps/scripts/db_population/parsers/common_parsers.py

```diff
@@ -88,11 +104,110 @@ def parse_telegram_name(tg_str: str) -> str | None:
     if match_result is not None and len(match_result.group(0)) == len(tg_str):
         return f"@{match_result.group(0)}"
 
-    raise ValueError(f"tg: {tg_str} is not valid")
+    return None
+
+
+def find_any_phone_number(number_str: str) -> list[str]:
+    number_str = number_str.replace(" ", "")
+    regex = re.compile(r"[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}")
+    return re.findall(regex, number_str)
+
+
+def parse_phone_number(number_str: str) -> str | None:
+    number_str = number_str.strip().rstrip()
+    country_codes_to_test = ["", "+380", "+7", "+1"]
```

Comment:
Note, this one does not matter, it can be anything, I think it is better to leave just one country code or use some +10000 just to distinguish it in the future